src/server/cluster/outgoing_slot_migration.cc

```diff
@@ -20,7 +20,10 @@
 #include "server/server_family.h"
 #include "util/fibers/synchronization.h"
 
-ABSL_FLAG(int, slot_migration_connection_timeout_ms, 5000, "Timeout for network operations");
+ABSL_FLAG(int, slot_migration_connection_timeout_ms, 2000,
+          "Connection creating timeout for migration operations");
+ABSL_FLAG(int, migration_finalization_timeout_ms, 30000,
+          "Timeout for migration finalization operation");
```

Comment:
Can we add a little bit more in the description about what these flags do because `Connection creating timeout for migration operations` seems vague. I would expect with that description a timeout would trigger at 2000 ms which would include the 30000 ms for the migration to finalize

Reply:
I guess we can always reiterate on the descriptions so I leave this for you and the rest of team to decide